<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1008</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1008</width>
    <height>630</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoliJump_Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resourses.qrc">
    <normaloff>:/Character/Resources/lolita/icon.png</normaloff>:/Character/Resources/lolita/icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>31</width>
    <height>28</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="HPLabel_Black">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>470</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resourses.qrc">:/HP/Resources/Black_HP.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="HPLabel_Red">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>215</y>
      <width>450</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Floor_Label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Recent Floors&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt; color:#ffffff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Platform_Label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Jumped platforms&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt; color:#ffffff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Time_Label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>270</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Game time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt; color:#ffffff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Record_Label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Highest Record&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:64pt; color:#ffffff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="Record" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="TopLaser_Label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>505</width>
      <height>17</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="Resourses.qrc">:/Bar/Resources/TopLaser.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Character_Label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start Game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resourses.qrc">
      <normaloff>:/Button/Resources/StartButton.png</normaloff>
      <normalon>:/Button/Resources/StartButton.png</normalon>
      <disabledoff>:/Button/Resources/StartButton_hover.png</disabledoff>
      <disabledon>:/Button/Resources/StartButton_hover.png</disabledon>
      <selectedoff>:/Button/Resources/StartButton_hover.png</selectedoff>
      <selectedon>:/Button/Resources/StartButton_hover.png</selectedon>:/Button/Resources/StartButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>111</width>
      <height>111</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>500</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clean The Record&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="icon">
     <iconset resource="Resourses.qrc">
      <normaloff>:/Button/Resources/ClearButton.png</normaloff>:/Button/Resources/ClearButton.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resourses.qrc"/>
 </resources>
 <connections/>
</ui>
